.babelrc

```diff
@@ -1,8 +1,7 @@
 {
+  "presets": ["env"],
```

Comment:
I think should be:
```json
["env", {
      "targets": {
        "node": "6"
      }
    }]
```

Reply:
Ah, I didn't really grasped this, babel is actually used to transpile for node?

Then my renaming with the ``lib`` folders really doesn't make sense, I will revert this and change it to your version with naming ``src``.

Reply:
babel transpile code for every `npm publish`
+1 for `lib` to `src`
maybe `build` to `lib`?

Reply:
Yes yes, of course also ``build`` back to ``lib``, didn't get the whole concept. :-)

Reply:
Done.